recipes/conda-index/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "conda-index" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b3ca5248a714d355dff29b1c07f4f5b3088e9f660815b19bb40f0379664c744c
+
+build:
+  script: {{ PYTHON }} -m pip install --no-build-isolation . -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - conda-index = conda_index.cli:cli
```

Comment:
Remove the entry point section (see upstream)

Reply:
oops, good catch. Thanks